storages/memory-storage/src/lib.rs

```diff
@@ -71,18 +75,30 @@ impl Store for MemoryStorage {
 #[async_trait(?Send)]
 impl StoreMut for MemoryStorage {
     async fn insert_schema(&mut self, schema: &Schema) -> Result<()> {
+        let created = HashMap::from([
+            ("OBJECT_TYPE".to_owned(), Value::Str("TABLE".to_owned())),
```

Comment:
This would not be necessary because which the same `OBJECT_TYPE` property is being used in execution layer.

Reply:
I removed OBJECT_TYPE in this PR but it may be needed if we implement metadata of index later since only OBJECT_NAME is used as a key here.
Just in case TABLE_NAME and INDEX_NAME are identical (but object_types are different), both OBJECT_NAME and OBJECT_TYPE should be a key